app/models/course.rb

```diff
@@ -1,4 +1,6 @@
 class Course < ApplicationRecord
+  belongs_to :course_group, optional: true
```

Comment:
```suggestion
  belongs_to :course_group
```

from @peteryates : we need every course in a group, as that's the only way it can be linked to a schedule